RetourPlugin.php

```diff
@@ -180,7 +180,7 @@ public function getReleaseFeedUrl()
      */
     public function getVersion()
     {
-        return '1.0.23';
+        return '1.0.23-locale-fix';
```

Comment:
I'm hoping I can just make another tag, and point the `1.0.23` release to use this tag. If that doesn't work, I'll just do another PR to change this to `1.0.24`